### ISSUE TITLE
[Question] Does nfs-subdir only support one path?

### ISSUE THREAD

**Issue by alteredtech** [kubernetes-sigs/nfs-subdir-external-provisioner#191]
Would I need a separate helm chart if I had another nfs share that I wanted to be mounted?

**Comment by yonatankahana** [comment]
yes, another installation with a different storageClass and provisionerName is required, read more in [Install Multiple Provisioners](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/tree/master/charts/nfs-subdir-external-provisioner#install-multiple-provisioners)
